USAGE.md

```diff
@@ -0,0 +1,274 @@
+# ECS Tooling Usage
+
+In addition to the published schema and generated artifacts, the ECS repo also contains tools to generate artifacts based on the current published and custom schemas.
```

Comment:
We should expand more here.

Q. Why do people care? 
A.

* ECS has too many fields
* ECS doesn't have their custom fields.

=> Users can use ECS tools to fix both of these things in their project. They can maintain only their customizations and use these tools to produce relevant artifacts for their various data sources.

Reply:
In addition to this, we could specifically call out the two artifacts that I expect users will find most useful. The Elasticsearch template is expected to be most useful for users in general. But Beats contributors may instead be interested in the Beats-formatted YAML field definition files.

Reply:
These are great additions. Maybe another nice, future addition is tutorial-like walkthroughs showing end-to-end how to define your own schema customizations, generate artifacts, and then application in Beats or ES.